app/views/microposts/new.js.erb

```diff
@@ -1,2 +1,5 @@
-alert("Hii please proceed..");
-$("#blogs").append("<%=j render 'form', item: @micropost%>");
+$(document).ready(function(){
+  alert("Hii please proceed..");
+  $("#blogs").append("<%= j render 'form', item: @micropost %>");
+  flatpickr('.flatpickr',{ enableTime: true, dateFormat: "Y-m-d H:i" })
```

Comment:
@HrisikeshSinghaChowdhury please add this inside application.js instead of repeating same code.

Reply:
i did that then the flatpickr was not working for new page